frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/users/UserUtils.java

```diff
@@ -202,18 +203,22 @@ public static void fillThriftUserFromThriftUser(final org.eclipse.sw360.datahand
     public static UserGroup getUserGroupFromLiferayUser(com.liferay.portal.model.User user) {
 
         try {
-            List<Role> roles = user.getRoles();
-
-            List<String> roleNames = new ArrayList<>();
-
-            for (Role role : roles) {
-                roleNames.add(role.getName());
+            List<String> roleNames = user.getRoles().stream()
+                    .map(Role::getName)
+                    .collect(Collectors.toList());
+
+            if (roleNames.contains(PortalConstants.ROLENAME_SW360_ADMIN)) {
+                return UserGroup.SW360_ADMIN;
+            } else if (roleNames.contains(PortalConstants.ROLENAME_CLEARING_ADMIN)) {
+                return UserGroup.CLEARING_ADMIN;
+            } else if (roleNames.contains(PortalConstants.ROLENAME_ECC_ADMIN)) {
+                return UserGroup.ECC_ADMIN;
+            } else if (roleNames.contains(PortalConstants.ROLENAME_SECURITY_ADMIN)) {
+                return UserGroup.SECURITY_ADMIN;
+            } else if (roleNames.contains(PortalConstants.ROLENAME_ADMIN)) {
+                return UserGroup.ADMIN;
```

Comment:
If a liferay user has more than one role, ADMIN should be preferred over all others.